rocketpy/Rocket.py

```diff
@@ -689,74 +694,44 @@ def finNumCorrection(n):
                 + (1 / 6) * (Cr + Ct - Cr * Ct / (Cr + Ct))
             )
 
-        # Calculate lift parameters for planar fins
         if not airfoil:
-
-            clalphaSingleFin = Function(
-                lambda mach: 2
-                * np.pi
-                * AR
-                * (Af / Aref)
-                / (2 + np.sqrt(4 + ((beta(mach) * AR) / (np.cos(gamac))) ** 2)),
+            # Defines clalpha2D as 2*pi for planar fins
+            clalpha2D = Function(lambda mach: 2 * np.pi / beta(mach))
+        else:
+            # Defines clalpha2D as the derivative of the
+            # lift coefficient curve for a specific airfoil
+            airfoilCl = Function(
+                airfoil[0], interpolation="linear", extrapolation="natural"
             )
+            if airfoil[1] == "degrees":
+                for i in range(len(airfoilCl)):
+                    airfoilCl[i][0] = np.radians(airfoilCl[i][0])
```

Comment:
Current implementation does not support Functions based on callables (Functions not created with lists or csv, but a lambda function or similar). This simple change can make it compatible with everything. What do you thing?

```suggestion
            if airfoil[1] == "degrees": airfoilCl *= np.pi/180 # Convert to radians
```

Reply:
This actually doesn't work for .csv or .txt. I added a `if callable(airfoil[0]):` make callables work

Reply:
- Really? It should work! What is the error that occurs? Must be a bug in the Function class.

Reply:
No errors, just weird values.
Using only `if airfoil[1] == "degrees": airfoilCl *= np.pi/180` :

![image](https://user-images.githubusercontent.com/69485049/154571706-5f428c64-adb4-430a-a360-ac8d3701cc0c.png)

Checking with callable:

![image](https://user-images.githubusercontent.com/69485049/154571895-6b5f3142-f4ef-4801-a655-6160ef9676f2.png)